NEW_RELEASE.md

```diff
@@ -34,11 +21,19 @@ So you think you are ready to make a release, eh? Follow the steps below :)
   - [ ] Update end-of-line tester with new release
   - [ ] Verify all tests function
 
-## Finalizing the release
-Alright so the release is tested and it actually works! Nice!
-- [ ] merge the prerelease branch into develop and merge that into the main branch
-- [ ] use poetry to bump the version `poetry version ${VERSION} && git add pyproject.toml && git commit -m "Bump release"`
-- [ ] use git to delete any prelease tags `git push origin -d $(git tag -l "${VERSION}-alpha*") && git tag -d $(git tag -l "${VERSION}-alpha*")`
-- [ ] Tag the changes so we can make a release on GitHub `git tag -as ${VERSION} -m '' && git push --follow-tags`
-- [ ] Make a release using the GitHub interface, add notes from the CHANGELOG.md
-- [ ] Make a new Raspberry Pi image to program onto units before shipping.
+## Versioning
+This project follows [Semantic Versioning](https://semver.org/). Here are some examples of versions we've used before:
+* `0.2.1`: this is a public general-availability bugfix release of the `0.2` feature release.
+* `0.3.0-alpha.0` is the first alpha release of the `0.3` feature release.
+
+## Making a release
+- [ ] Ensure the PR(s) with your features & fixes are merged into `main`.
+- [ ] Check out and pull `main`
+- [ ] Update the changelog with notes and the latest version (and commit it!)
```

Comment:
This reads as if the changelog/notes should be committed directly onto main, but that's not possible right?

Reply:
ope, good catch. I think the latest rendition of this oughta be "more correct".